testing.Rmd

```diff
@@ -0,0 +1,14 @@
+---
+title: "Test Example"
+output: github_document
+---
+
+```{r}
+
+library(tibbletest)
```

Comment:
can you move this over to the README and remove these two files?  the README doesn't need to be fully fleshed out quite yet, but I think it makes more sense there